internal/services/databricks/registration.go

```diff
@@ -44,14 +43,10 @@ func (r Registration) SupportedDataSources() map[string]*pluginsdk.Resource {
 func (r Registration) SupportedResources() map[string]*pluginsdk.Resource {
 	resources := map[string]*pluginsdk.Resource{
 		"azurerm_databricks_workspace":                                resourceDatabricksWorkspace(),
-		"azurerm_databricks_workspace_customer_managed_key":           resourceDatabricksWorkspaceCustomerManagedKey(), // TODO: Remove in 4.0
```

Comment:
This is what I'm referring to in the comment above, so this would technically be a breaking change now.

Reply:
update comments & flags to 5.0 for resource